db/migrations/0002_cinemahall_movie_moviesession.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.0.2 on 2024-10-23 20:40
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CinemaHall',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('rows', models.IntegerField()),
+                ('seats_in_row', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Movie',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=255)),
+                ('description', models.TextField()),
+                ('actors', models.ManyToManyField(related_name='actor', to='db.Actor')),
+                ('genres', models.ManyToManyField(related_name='genre', to='db.Genre')),
```

Comment:
The `related_name` 'genre' for the `genres` field might be misleading. It should be pluralized to 'genres' to better reflect that this is a reverse relation for multiple genres associated with a movie.